scripts/pptx_format.py

```
@@ -233,16 +236,16 @@
     Apply formatting profile to slide XML data.
-    
+
     Args:
         slide_xml_data: Raw slide XML bytes
-        
+
     Returns:
         Formatted slide XML bytes
     """
-    root = ET.fromstring(slide_xml_data)
+    root = safe_fromstring(slide_xml_data)
```

Ruff (0.13.1)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)